contribute/style-guides/documentation-markdown-guide.md

```diff
@@ -12,34 +12,35 @@ HTML heading levels:
 
 **Example**
 
-* \# is \<h1>.
-* \#\# is \<h2>.
-* \#\#\# is \<h3>.
+- \# is \<h1>.
+- \#\# is \<h2>.
+- \#\#\# is \<h3>.
 
 Start your document with a single ``#`` for the title of the page. Add the sub-headings with two ``##``.
 
 ## Bold and emphasis
 
-* Make text **bold** using two asterisks.
+- Make text **bold** using two asterisks.
 
 **Example:** It is ``**important**`` to use GitHub Flavored Markdown emoji consistently.
 
-* Make text ``_emphasized_`` using single `` _underscores_``. Do not use the single asterisk, it can be easily confused with bold.
+- Make text ``*emphasized*`` using single `` _underscores_`` or a single asterisk.
 
-**Example:** GitHub-flavored markdown emoji should _only_ appear in specific cases.
+**Example:** GitHub Flavored Markdown emoji should _only_ appear in specific cases.
 
 
 ## Links and references
 
-Create links to other website by wrapping the display text in square brackets, and the web URL in curved brackets.
+Create links to other website by wrapping the display text in square brackets, and
```

Comment:
Please do not make weird line breaks. It's easier to edit sentences when they're not broken into multiple lines.